follow_human:
  ros__parameters:
    wait_msg_timeout_secs: 1.0
    angular_vel_multiplier: 0.7
    forward_vel: 0.1
    human_lost_angular_vel: 0.5
    min_dist_threshold: 0.1
    filter_value: 0.9
